CHANGES/6800.bugfix.rst

```diff
@@ -0,0 +1 @@
+Modified websocket ``receive_str`` and ``receive_bytes`` methods to raise new ``WSMessageTypeError`` exception, instead of generic ``TypeError``, when websocket messages of incorrect types are received -- by :user:`ara-25`.
```

Comment:
```suggestion
Modified websocket ``receive_str`` and ``receive_bytes`` methods to raise new :py:exc:`~aiohttp.WSMessageTypeError``` exception, instead of generic ``TypeError``, when websocket messages of incorrect types are received -- by :user:`ara-25`.
```

``TypeError``, ``receive_str`` and ``receive_bytes`` could also get references

Reply:
Updated